consensus/metrics/init.go

```diff
@@ -0,0 +1,48 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	namespace = "lightchain"
+	metricsSubsystem = "consensus"
+)
+
+var (
+	moduleAbciConstLabelValues = map[string]string{"module": "abci"}
+)
+
+const errorCodeLabel = "error_code"
+
+// Metrics contains metrics exposed by this package.
+type Metrics struct {
+	CheckTxsTotal       CheckTxTotalMetric
+	CheckErrTxsTotal    CheckErrTxsTotalMetric
+	DeliverTxsTotal     DeliverTxsTotalMetric
+	DeliverErrTxsTotal  DeliverErrTxsTotalMetric
+	CommitBlockTotal    CommitBlockTotalMetric
+	CommitErrBlockTotal CommitErrBlockTotalMetric
+}
+
+func NewMetrics(registry *prometheus.Registry) Metrics {
+	return Metrics{
```

Comment:
keep the pointer here please because this obj is constantly changing as all of its attributes are pointers as well to make it explicit to developers. Same in db pkg.